src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -66,6 +70,17 @@ public void parseCommand_addCto() throws Exception {
         assertEquals(new AddContactToCommand(projectIndex, person), command);
     }
 
+    @Test
+    public void parseCommand_addDto() throws Exception {
```

Comment:
might be better to name the test parseCommand_addDto_success since we  are only testing the success scenario here. As per https://se-education.org/guides/conventions/java/intermediate.html, we should only leave our the expected behaviour if we are testing for all scenarios.

Reply:
Good idea!